app/browser/dataset-preview.tsx

```diff
@@ -166,9 +180,15 @@ export const DataSetPreview = ({
               className={classes.createChartButton}
               component="a"
             >
```

Comment:
@noahonyejese you probably want `target="_top"` here, otherwise only the page within the iframe will navigate.  same in the other `Create visualization on visualize from this dataset` link below…